docs/DeveloperGuide.md

```diff
@@ -238,6 +238,27 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### Customise Order Feature
+
+#### Implementation
+
+This feature has 2 crucial methods for this feature are in the `CustomiseCommand` and `PersonCard` classes.
+
+* `CustomiseCommand#changeAttributeOrder(Model model)` - saves the new attribute order in the `preferences.json`
```

Comment:
`in the preferences.json` can be `in preferences.json` or `in the preferences.json file`

Reply:
Ok, I have changed it to `in the preferences.json file`.